src/Nervos/InputSelector.cpp

```diff
@@ -0,0 +1,185 @@
+// Copyright © 2017-2022 Trust Wallet.
+//
+// This file is part of Trust. The full Trust copyright notice, including
+// terms governing use, modification, and redistribution, is contained in the
+// file LICENSE at the root of the source code distribution tree.
+
+#include "InputSelector.h"
```

Comment:
There seems to be a huge amount of code duplication (from Bitcoin folder), this should be reduced

Reply:
Yes, removed this and implemented logic as per Neuron wallet (Nervos's official wallet).